### ISSUE TITLE
Decode Bitmap returns wrong values if the first bit is false.

### ISSUE THREAD

**Issue by ssozonoff** [thesolarnomad/lora-serialization#13]
message
  .addBitmap(false, true, false, true, false, false, false, false)

Returns less than 8 flags and the values seem to be offset.

**Comment by joscha** [comment]
Thanks for the report @ssozonoff - I am on mobile right now - can you write a failing test case and/or give an example?

**Comment by ssozonoff** [comment]
Hi,

As far as I am concerned the below does not deserialise correctly TTN side.
There are only 6 flags returned from a-f

Setting the first value to true and we get flags for a-h

```
message
  .addBitmap(false, true, false, false, false, false, false, false)
  .addTemperature(waterTempRunningAvg.getAverage())
  .addTemperature(batteryVoltage);  
```

**Comment by joscha** [comment]
@ssozonoff sorry about the long wait, this is now fixed and released in lora-serialization@3.0.1
